src/sniffer.c

```diff
@@ -2282,25 +2282,17 @@ static int SetupKeys(const byte* input, int* sslBytes, SnifferSession* session,
     /* Static DH Key */
     if (ksInfo && ksInfo->dh_key_bits != 0 && keys->dhKey) {
         DhKey dhKey;
+#ifdef HAVE_PUBLIC_FFDHE
         const DhParams* params;
         word32 privKeySz;
+#else
+        word32 privKeySz = 0, p_len = 0;
+#endif
```

Comment:
the distinction between them (`privKeySize` initialized in one, not in the other) is substantive, so this needs to stay as-is.  otherwise we run the risk of analyzers reporting an unused assignment.